game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1567,15 +1589,7 @@ public void keyPressed(final KeyEvent e) {
         }
         // I for info
         if (keyCode == KeyEvent.VK_I || keyCode == KeyEvent.VK_V) {
-          String unitInfo = "";
-          if (unitsBeingMousedOver != null && !unitsBeingMousedOver.isEmpty()) {
-            final Unit unit = unitsBeingMousedOver.get(0);
-            final UnitAttachment ua = UnitAttachment.get(unit.getType());
-            if (ua != null) {
-              unitInfo = "<b>Unit:</b><br>" + unit.getType().getName() + ": "
-                  + ua.toStringShortAndOnlyImportantDifferences(unit.getOwner(), true, false);
-            }
-          }
+          String unitInfo = getUnitInfo();
```

Comment:
Now that this calls a method, could you declare this variable final?

Reply:
Done.